TUM Campus App/MVGManager.swift

```diff
@@ -23,16 +29,36 @@ final class MVGManager: Manager, SimpleSingleManager {
         self.config = config
     }
     
+    func performRequest(maxCache: CacheTime) -> Promise<[Station], APIError> {
+        return config.mvg.doObjectsRequest(
+            to: .getNearbyStations,
+            queries: ["latitude" : location.coordinate.latitude, "longitude" : location.coordinate.longitude],
+            at: ["locations"],
+            maxCacheTime: maxCache).map { $0.sorted(byLocation: \.location)
+        }
+    }
+    
     func fetchSingle() -> Response<DataElement?> {
-        return fetch().flatMap { stations in
+        return prefetch().flatMap { stations in
             guard let station = stations.first else { return .successful(with: nil) }
             return self.fetch(for: station).map { departures in
                 return DetailedStation(station: station, departures: departures)
             }
         }
     }
     
-    func fetch() -> Response<[Station]> {
+    func fetch() -> Response<[DetailedStation]> {
+        return prefetch().flatMap { stations in
+            return stations.flatMap { station in
```

Comment:
flatMap() is deprecated. Check if there’s another way to accomplish the desired functionality.